mix/ConstantCompilationControl.cpp

```diff
@@ -34,15 +34,15 @@
 
 using namespace dev::mix;
 
-ConstantCompilationControl::ConstantCompilationControl(AppContext* _context): Extension(_context, ExtensionDisplayBehavior::Tab)
+ConstantCompilationControl::ConstantCompilationControl(AppContext* _context): Extension(_context, ExtensionDisplayBehavior::HeaderView)
 {
 	connect(_context->codeModel(), &CodeModel::compilationComplete, this, &ConstantCompilationControl::update);
-	connect(_context->codeModel(), &CodeModel::compilationComplete, this, &ConstantCompilationControl::update);
+	_context->appEngine()->rootContext()->setContextProperty("constantCompilation", this);
```

Comment:
I think now is a good time to rename it to something like StatusPane as it is going to display all kinds of information, not just compilation results.